content/en/platform/corda/5.1/reference/corda-cli/preinstall.md

```diff
@@ -91,21 +91,21 @@ table th:nth-of-type(2) {
 | Argument | Description                                                                             |
 | --------------------------------------- | ---------------------------------------------------------|
 | \<path\> | The YAML file containing the Kafka, SASL, and TLS configurations. |
-| \-n, \-\-namespace=<namespace> | The namespace in which to look for the Kafka secrets if TLS or SASL is enabled. |
-| \-t, \-\-timeout=<timeout> | The timeout in milliseconds for testing the Kafka connection - defaults to 3000. |
+| \-n, \-\-namespace=<namespace\> | The namespace in which to look for the Kafka secrets if TLS or SASL is enabled. |
```

Comment:
don't you need an escape character for `<` too ?

Reply:
@yarivus no, it works, please see the preview: https://anna.preview.docs.r3.com/en/platform/corda/5.1/reference/corda-cli/preinstall.html#preinstall

Reply:
OK thanks 👍